### DISCUSSION TITLE
Share your setup!

### DISCUSSION THREAD

**Discussion by doitandbedone** [doitandbedone/ispyagentdvr-docker#148] [Show and tell]
Welcome to the community! Hope you are enjoying iSpy! Thought it be nice to share your setup with each other to give a better idea on how people are using iSpy. Here's a template for reference:

- Number of cameras
- Type of camera (Wifi, PoE, Ethernet, PTZ)
- Usage (Pet surveillance, outdoor)
- Brand/Model
- Features you like to use (Camera or iSpy)
- Tips/Hacks you have found

Have fun and thanks for sharing!

**Comment by doitandbedone** [comment]
Here's my setup:
- Have a total of 11 cameras
- 3 Wifi cameras for in doors/pet surveillance. One PTZ used to follow kitty activities.
- 8 PoE for outdoor surveillance.
- They are all different models but predominantly Amcrest brand. 
- It's nice to stitch them together and see them in VR!

**Comment by kysdaddy** [comment]
I found this thread and since I have two systems up and running I can add info here, but I have a question as well about the docker container size.

System 1 Supermicro server running unraid

108 TB of total storage 
56 TB available

Running 
plex
Home assistant
Unifi
Nextcloud
and a bunch more

Agent DVR docker 1.25GB
6 PTZ Wifi Off brand from amazon camhi default software and some use Icsee
9 Stationary Wifi cameras ff brand from Amazon Camhidefault software and some use Icsee

System 2 Dell Dell R820 running unraid

12 TB of total storage 
10.3 TB available

Running 
plex
Home assistant
Unifi

Agent DVR docker 15.7GB that's not a typo 15.7GB
4PTZ Wifi Off brand from amazon camhi default software 
22 Stationary Wifi cameras off-brand from Amazon Camhi default software 

My question is should the 2nd system's docker be that much larger with 11 more cameras?
